packages/next/src/client/image.tsx

```diff
@@ -513,6 +517,8 @@ const Image = forwardRef<HTMLImageElement | null, ImageProps>(
       onLoadingComplete,
       placeholder = 'empty',
       blurDataURL,
+      // @ts-expect-error - TODO: upgrade to `@types/react@18`
+      fetchPriority,
```

Comment:
Should this one be lowercase as well?

Reply:
This was to cover the case if the user set `<Image fetchPriority="low" />`. 

The public API is camelCase because thats how React normally does it, but the DOM attribute all lowercase. 

I can add a test for that to make sure its working.

Reply:
Added in [601402b](https://github.com/vercel/next.js/pull/47222/commits/601402b79cedbdcec12742e0ae9826dfab51e426)